.github/workflows/autobuild.yml

```diff
@@ -213,8 +213,8 @@ jobs:
           path: |
             C:\Qt
             C:\ChocoCache
-            ~\windows\NSIS
-            ~\libs\ASIOSDK2
+            ${{ github.workspace }}\windows\NSIS
```

Comment:
I probably didn’t take enough care about windows as I moved the SDK. Thanks for fixing

Reply:
I'm not even sure what broke it, maybe it had never worked correctly, although I vaguely remember putting lots of effort into the testing. Anyway, now it should be good.